.github/workflows/build_push_staging.yml

```diff
@@ -0,0 +1,93 @@
+name: Build and Push to Container Registry Staging
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - "src/**"
+      - "pom.xml"
+      - "docker/**"
+      - ".github/workflows/build_push_staging.yml"
+
+env:
+  GITHUB_SHA: ${{ github.sha }}
+  REGISTRY: 992382783569.dkr.ecr.ca-central-1.amazonaws.com
+
+permissions:
+  id-token: write
+  contents: write
+
+jobs:
+  build:
+    if: github.repository == 'cds-snc/dto-feedback-cj'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Audit DNS requests
+        uses: cds-snc/dns-proxy-action@main
+        env:
+          DNS_PROXY_FORWARDTOSENTINEL: "true"
+          DNS_PROXY_LOGANALYTICSWORKSPACEID: ${{ secrets.LOG_ANALYTICS_WORKSPACE_ID }}
+          DNS_PROXY_LOGANALYTICSSHAREDKEY: ${{ secrets.LOG_ANALYTICS_WORKSPACE_KEY }}
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 1.8
```

Comment:
How come you have all these extra steps in the staging as opposed to the production build? it could be legitimate, but I just wanted to ask.